README.md

```diff
@@ -13,6 +13,8 @@ $ sudo apt-get install postgresql-12 postgresql-server-dev-12
 $ sudo -u postgres createuser -P -e test  --interactive
 $ createdb -U test -W 360givingdatastore
 
+Note: Special Postgresql extensions are used for indexes, if you don't want to have these installed in the database or the database user doesn't have permissions to install extensions then set environment variable `SKIP_SPECIAL_DB_INDEX=true` before you run migrate.
```

Comment:
Does the env var need to be set when running `manage.py test`? If so, mention, otherwise looks good.